benchmarking/frameworks/caffe2/caffe2.py

```diff
@@ -341,13 +341,16 @@ def _processDelayData(self, data):
             lambda: collections.defaultdict(list))
         pattern = re.compile(r"^ID_(\d+)_([a-zA-Z0-9]+)_[\w/]+")
         for d in data:
+            print(d)
             for k, v in d.items():
                 for kk, vv in v.items():
                     key = k + " " + kk
                     details[key]["values"].append(vv)
                     details[key]["type"] = k
                     # although it is declared as list
                     details[key]["metric"] = kk
+                    # TODO: add unit for each metric in observer
+                    details[key]["unit"] = "ms"
```

Comment:
I'd rather multiply 1000 on the caffe2_benchmark side. Otherwise we will get lots of zeros, since we no longer multiply 1000 on the harness side. Then we change the unit to be us. What do you think?

Reply:
Current format is still not extensible.